types/atom/index.d.ts

```diff
@@ -264,10 +287,10 @@ export interface CommandRegistryTargetMap extends HTMLElementTagNameMap {
 }
 
 export type CommandRegistryListener<TargetType extends EventTarget> = {
-  didDispatch(event: CommandEvent<TargetType>): void,
-  displayName?: string,
-  description?: string,
-  hiddenInCommandPalette?: boolean,
+    didDispatch(event: CommandEvent<TargetType>): void | Promise<void>,
+    displayName?: string,
+    description?: string,
+    hiddenInCommandPalette?: boolean,
 } | ((event: CommandEvent<TargetType>) => void);
```

Comment:
Uhhh... sorry, but this line probably should also be `void | Promise<void>`, since it's just a shorthand for setting *only* `didDispatch`.